cubedash/_pages.py

```diff
@@ -35,6 +35,8 @@
 _HARD_SEARCH_LIMIT = app.config.get("CUBEDASH_HARD_SEARCH_LIMIT", 150)
 _DEFAULT_GROUP_NAME = app.config.get("CUBEDASH_DEFAULT_GROUP_NAME", "Other Products")
 
+_DEFAULT_ARRIVALS_DAYS: int = app.config.get("CUBEDASH_DEFAULT_ARRIVALS_DAY_COUNT", 14)
```

Comment:
Is there a way to make Flask app configs self-describing. There seem to be more and more knobs in explorer 💯

Reply:
Not that I've seen. The current method is to [mention them all in the README](https://github.com/opendatacube/datacube-explorer#how-can-i-configure-the-deployment)

( .... which I haven't done here. Give me 30 seconds :slightly_smiling_face:  )

Reply:
Added